packages/x-components/src/components/layouts/fixed-header-and-asides-layout.vue

```diff
@@ -108,8 +109,8 @@
   })
   export default class FixedHeaderAndAsidesLayout extends mixins(LayoutsMixin) {
```

Comment:
I don't know why, if we let these two properties type to be inferred, it adds a MainScroll type there, which is incorrect and very strange.
![image](https://user-images.githubusercontent.com/68222542/210322416-58d35add-7657-4fa5-91cb-12f26762b565.png)

![image](https://user-images.githubusercontent.com/68222542/210322331-925f08e6-2615-4477-af92-aaeaf7d07d75.png)